UnivaPay-for-WooCommerce/tests/e2e/models/univapay/inline-form.ts

```diff
@@ -0,0 +1,31 @@
+import { Selector } from "testcafe"
+import { MockCardData } from "../../helper/mock"
+
+export class UnivapayInlinePage {
+    iframe: Selector
+    email: Selector
+    name: Selector
+    card: Selector
+    expiry: Selector
+    cvv: Selector
+
+    constructor() {
+        this.iframe = Selector('iframe[name="univapay-checkout-connector"]', { timeout: 3000 })
+        this.email = Selector('input[name="data.email"]')
+        this.name = Selector('input[name="data.cardholder"]')
+        this.card = Selector('input[name="data.cardNumber"]')
+        this.expiry = Selector('input[name="data.exp"]')
+        this.cvv = Selector('input[name="data.cvv"]')
+    }
+
+    async fillInlineForm(t: TestController, mockCardData: MockCardData) {
+        await t
+            .switchToIframe(this.iframe)
+            .typeText(this.name, mockCardData.cardName)
+            .typeText(this.card, mockCardData.cardNumber)
+            .typeText(this.expiry, mockCardData.expiry)
+            .typeText(this.cvv, mockCardData.cvv)
+            .switchToMainWindow()
+    }
```

Comment:
```suggestion
    private iframe = Selector('iframe[name="univapay-checkout-connector"]', { timeout: 3000 });
    private selectors = {
        email: Selector('input[name="data.email"]'),
        name: Selector('input[name="data.cardholder"]'),
        card: Selector('input[name="data.cardNumber"]'),
        expiry: Selector('input[name="data.exp"]'),
        cvv: Selector('input[name="data.cvv"]')
    };

    async fillInlineForm(mockCardData: MockCardData) {
        await t
            .switchToIframe(this.iframe)
            .typeText(this.selectors.name, mockCardData.cardName)
            .typeText(this.selectors.card, mockCardData.cardNumber)
            .typeText(this.selectors.expiry, mockCardData.expiry)
            .typeText(this.selectors.cvv, mockCardData.cvv)
            .switchToMainWindow();
    }
```

Reply:
nitpicks: may be, we could reduce codes boilerplates by eliminating the constructor (if it doesn't take any arguments). Same for other places.